packages/shared-types/src/UtilityTypes.ts

```diff
@@ -56,4 +56,4 @@ export type CSSShorthandValue<Value extends string> =
 export type OtherHTMLAttributes<
   Props,
   Attributes extends React.HTMLAttributes<Props> = React.AllHTMLAttributes<Props>
-> = Nullable<Omit<Attributes, keyof Props>>
```

Comment:
This had to be changed to be not nullable because when we have a component which uses the `getElementType` utility, the returned element from the `getElementType` could be a native DOM element, which essentially has the same typing as `React.HTMLAttributes` but without the attributes on it being nullable.

So what that means is when we have a scenario for example with the `View` component:

```tsx

render() {
  ...
     <View as="div" className={null}>some content</View>
  ...
```
The rendered element in the DOM will be a `div` element and its attributes can only take value types or `undefined` but not the value `null`.

Reply:
Seems nice!